File: src/abagile/aoc/2023/template.clj
(ns abagile.aoc.2023.dayXX
  (:require
   [abagile.aoc.util :as util]))

(def input (->> (util/read-input-split "2023/dayXX.txt" #"\n")))

(defn part1
  []
  (time (->> input)))


(defn part2
  []
  (time (->> input)))

(defn -main [& _]
  (println "part 1:" (part1))
  (println "part 2:" (part2)))

(comment
  (-main))

File: ts/2025/template.ts
let part1 = 0, part2 = 0
for await (const line of console) {
	if (line === "") continue
	console.log(line)
}
console.log("The result for part 1: %d", part1)
console.log("The result for part 2: %d", part2)
